featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -936,7 +939,16 @@ def update_feature_columns(feature_data, data):
         data.update(new_cols)
         return data
 
-    return data.assign(**new_cols)
+    if isinstance(data, pd.DataFrame):
+        return pd.concat([data, pd.DataFrame(new_cols, index=data.index)], axis=1)
+    else:
```

Comment:
Styling question: Should we have an `else` block here?

Reply:
Doesn't seem necessary. I will remove.